database-commons/src/main/java/io/cdap/plugin/db/connector/AbstractDBSpecificConnectorConfig.java

```diff
@@ -42,6 +48,12 @@ public abstract class AbstractDBSpecificConnectorConfig extends AbstractDBConnec
   @Nullable
   protected Integer port;
 
+  @Name(ConnectionConfig.TRANSACTION_ISOLATION_LEVEL)
+  @Description("The transaction isolation level for the database session.")
+  @Macro
+  @Nullable
+  protected String transactionIsolationLevel;
```

Comment:
isn't this class also used by `SQLServer`?

Reply:
Yes, it is. Should we hide the property from `SQLServer` Plugins for now? OR 
add this property to the respective connector configuration classes of `PostgreSQL` and `MySQL` plugins and remove from parent connector config?

Reply:
Added [here](https://github.com/data-integrations/database-plugins/pull/583/commits/e3b56c30ea7dcac93496a1da1491fcb6c792fb05#diff-d571ef313983e152bfbb7bd2241079707b3df44b3f0d0e8d1879ea84b4d87aea)